data/views/node.2.0.json

```diff
@@ -3,21 +3,28 @@
   "createdAt": "<%=createdAt%>",
   "id" : "<%=id%>",
   "name": "<%=name%>",
-<% if (hasOwnProperty("identifiers")) { %>
-  "identifiers": <%- JSON.stringify(identifiers) %>,
+  <% if (hasOwnProperty("identifiers")) { %>
+"identifiers": <%- JSON.stringify(identifiers) %>,
 <% } else  {%>
-  "identifiers": [],
+"identifiers": [],
 <% } %>
 <% if (hasOwnProperty("tags")) { %>
-   "tags": <%- JSON.stringify(tags) %>,
+"tags": <%- JSON.stringify(tags) %>,
 <% } else  {%>
-  "tags": [],
+"tags": [],
 <% } %>
 <% if (hasOwnProperty("obmSettings")) { %>
-  "obmSettings": <%- JSON.stringify(obmSettings) %>,
+"obmSettings": <%- JSON.stringify(obmSettings) %>,
 <% } else  {%>
-  "obmSettings": [],
+"obmSettings": [],
 <% } %>
-  "type": "<%=type%>",
-  "updatedAt": "<%=updatedAt%>"
-}
+"type": "<%=type%>",
+<% if (hasOwnProperty("sku")) { %>
+<% if (sku === null) { %>
+"sku" : null,
+<% } else  {%>
+"sku" : "<%=basepath %>/skus/<%=sku%>",
```

Comment:
@yyscamper: Consistency with the 1.1 API has never been a goal of the 2.0 design.  By semver rules (and our own) we have the flexibility to develop new conventions if we feel they are appropriate.  Feedback on this is welcome but I don't want to operate in a mode where every change needs core committer pre-approval.  Our approach has been to implement the new API based on our best judgement and collect feedback on our work in Sprint and PR reviews.  IMO - comments like this should include a suggested alternative with an explanation of how that alternative improves things.
.
@anhou You are right that this is inconsistent with other parts of the 2.0 API where views have not been implemented yet.  There are stories in our backlog to implement views for other routes (workflows, pollers, etc).  In doing so, we'll make sure that we use a consistent set of conventions unless we have a reason to do otherwise.  For example, there are inconsistencies between /api/2.0/workflows and /api/2.0/nodes/{identifier}/workflows that we plan to change.